src/berry_tag_screen.c

```diff
@@ -143,11 +143,11 @@ static const struct WindowTemplate sWindowTemplates[] =
 
 static const u8 *const sBerryFirmnessStrings[] =
 {
-    gBerryFirmnessString_VerySoft,
-    gBerryFirmnessString_Soft,
-    gBerryFirmnessString_Hard,
-    gBerryFirmnessString_VeryHard,
-    gBerryFirmnessString_SuperHard
+    COMPOUND_STRING("Very soft"),
+    COMPOUND_STRING("Soft"),
+    COMPOUND_STRING("Hard"),
+    COMPOUND_STRING("Very hard"),
+    COMPOUND_STRING("Super hard")
```

Comment:
Can you add `BERRY_FIRMNESS_` indexes to this table?

Unrelated, but it seems that this table doesn't consider `BERRY_FIRMNESS_UNKNOWN`, which might cause issues for Kee and Maranga berries. She should check them.

Reply:
Do you mean something like
`[BERRY_FIRMNESS_VERY_SOFT] = COMPOUND_STRING("Very soft"),`?

If we add BERRY_FIRMNESS_UNKNOWN, then I think we'll need to update src/berry_tag_screen.c to change `sBerryFirmnessStrings[berry->firmness - 1]` to `sBerryFirmnessStrings[berry->firmness]` (mainly a note for me so I don't forget tomorrow)

Reply:
Makes sense to me. It'd be better than to have them underflow imo.

Reply:
Done

Reply:
You didn't update the call to `sBerryFirmnessStrings`.

Reply:
Oops. Now it's updated